packages/runtime-plugin-hmr/src/index.ts

```diff
@@ -32,7 +32,7 @@ export default <FarmRuntimePlugin>{
       });
 
       socket.addEventListener('open', () => {
-        console.log('[Farm HMR] connected to the server');
+        console.log(`[Farm HMR] connected to the server  PORT...${port} `);
       });
```

Comment:
Printing in a browser should not require a port number

Reply:
ok i delete it